[no-land] try fix 1

smessmer opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26482 [wip] Error checking for KernelFunction::callUnboxed()
* #30712 Better error messages thanks to compile time type names
* #31307 [no-land] try fix 2
* **#31306 [no-land] try fix 1**
* #26628 Remove CAFFE_KNOWN_TYPE

Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

smessmer committed:
[no-land] try fix 1



Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit cfc0e14:
* **2/2** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=cfc0e145cc466c555bc73e6a5bcb232310c6fb5c).

### 1 new failure recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3960392) pytorch_windows_test1 (1/1)
**Step:** "Test" ([details](https://dr.pytorch.org/build-details.html?build_id=74856271))
<details>
<summary>
<code> - cmake (exited 1) - cmake not installed. An error occurred during installation:</code>
</summary>

```
 The remote server returned an error: (503) Server Unavailable. Service Unavailable 
cmake package files install completed. Performing other installation steps. 
The install of cmake was NOT successful. 
cmake not installed. An error occurred during installation: 
 The remote server returned an error: (503) Server Unavailable. Service Unavailable 
 
Chocolatey installed 0/1 packages. 1 packages failed. 
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log). 
 
Failures 
 - cmake (exited 1) - cmake not installed. An error occurred during installation: 
 The remote server returned an error: (503) Server Unavailable. Service Unavailable 
```

</details>

### 1 failure *not* recognized by patterns:
| Job | Step | Status |
| --- | --- | --- |
| ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) <sup>pytorch_windows_test2</sup> | <sup>Test</sup> | New in PR |


---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]

smessmer committed:
Update on "[no-land] try fix 1"


Differential Revision: [D19085561](https://our.internmc.facebook.com/intern/diff/D19085561/)

[ghstack-poisoned]
